README.md

```diff
@@ -20,9 +20,31 @@ WIKI: https://wiki.mozilla.org/Idea-Town
 
 * [Install Docker](http://docs.docker.com/mac/started/) and docker-compose
 
-* `docker up`
+* Make sure you have a default Docker machine, i.e.:
+  * `docker-machine create -driver virtualbox default`
 
-* Visit http://0.0.0.0:8000
+* Make sure your shell can see the default Docker machine, i.e.:
+  * `eval "$(docker-machine env default)"`
+
+* Create & set up the Docker containers:
+  * `docker-compose up`
+  * This may take some time.
+
+* To see the IP address of the default Docker machine:
+  * `docker ip default` 
```

Comment:
Yeah, that should be `docker-machine ip default`. And this is the version I'm using, freshly installed from the Docker site: 
```
➜  idea-town git:(moar-docker) ✗ docker version
Client:
 Version:      1.8.1
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   d12ea79
 Built:        Thu Aug 13 02:49:29 UTC 2015
 OS/Arch:      darwin/amd64

Server:
 Version:      1.8.1
 API version:  1.20
 Go version:   go1.4.2
 Git commit:   d12ea79
 Built:        Thu Aug 13 02:49:29 UTC 2015
 OS/Arch:      linux/amd64
```